python/py-mkl/Portfile

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-mkl
+version             2019.0
+revision            0
+platforms           darwin
+supported_archs     noarch
```

Comment:
This is correct for the py-mkl stub port, but it's inherited by the subports which should be x86_64.